docs/releases/4.0.txt

```diff
@@ -217,7 +217,9 @@ Management Commands
 Migrations
 ~~~~~~~~~~
 
-* ...
+* The ``makemigrations`` command might generate a no-op ``AlterField``
+  operation in some rare cases.
```

Comment:
Thanks for that! I guess we could mention that this should only happen for `ForeignKey` fields unless we're aware it happens for other options.